src/index.html

```diff
@@ -5,8 +5,393 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="defalit"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&family=Nothing+You+Colid+Do&family=Roboto:ital,wght@0,500;0,700;1,500;1,700&family=Satisfy&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="moyo_logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__list-link nav__list-link--is-active is-active" href="#apple">Apple</a>
+          </li>
+
+          <li>
+            <a class="nav__list-link" href="#samsung">Samsung</a>
+          </li>
+
+          <li>
+            <a class="nav__list-link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li>
+            <a
+              class="nav__list-link"
+              href="#laptopsAndComputers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a class="nav__list-link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li>
+            <a class="nav__list-link" href="#tablets">Tablets</a>
+          </li>
+
+          <li>
+            <a class="nav__list-link" href="#photo">Photo</a>
+          </li>
+
+          <li>
+            <a class="nav__list-link nav__list-link--last" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+              src="images/imac.jpeg"
+              alt="imac_image"
+              class="card__image"
+          >
+
+          <h1 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
```suggestion
          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```
h1 can be only one per page